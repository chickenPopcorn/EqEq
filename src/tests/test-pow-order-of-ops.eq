Foo = {
 a = { 2; }
 b = { 3; }
}

Foo:find a {
  print("a ^ b ^ 2 = %.0f\n", a ^ b ^ 2);
}
