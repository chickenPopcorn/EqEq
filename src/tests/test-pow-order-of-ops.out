a ^ b ^ 2 = 64
